src/App/Fossa/Analyze/Upload.hs

```diff
@@ -111,10 +111,7 @@ uploadSuccessfulAnalysis (BaseDir basedir) metadata jsonOutput revision scanUnit
 
     if (orgSupportsReachability org)
       then void $ upload revision metadata reachabilityUnits
-      else
-        unless (null reachabilityUnits) $
-          logDebug . pretty $
-            "Organization: (" <> show (organizationId org) <> ") does not support reachability! skipping reachability analysis upload!"
+      else logDebug . pretty $ "Organization: (" <> show (organizationId org) <> ") does not support reachability! skipping reachability analysis upload!"
```

Comment:
[optional] I love more personality in my tools but I've firsthand witnessed corporate users demeaning tools that aren't, well, dry and corporate.

If the goal is emphasis using color or font weighting doesn't, IME, trigger these kinds of issues.
```suggestion
      else logDebug . pretty $ "Organization: (" <> show (organizationId org) <> ") does not support reachability. Skipping reachability analysis upload."
```